frontend/packages/schema/src/parser/sql/postgresql/processSqlInChunks.ts

```diff
@@ -281,12 +296,14 @@ export const processSQLInChunks = async (
   chunkSize: number,
   callback: (
     chunk: string,
+    chunkOffset?: number,
```

Comment:
Shouldn't this also be required?
ref: https://github.com/liam-hq/liam/pull/3118#discussion_r2293093507

Reply:
This one too!
https://github.com/liam-hq/liam/pull/3118/commits/1f21a131f261a6efcd8d236966f41d22737d819e